spec/controllers/publics_controller_spec.rb

```diff
@@ -42,7 +42,7 @@
       post :receive, "guid" => @user.person.guid.to_s, "xml" => xml
     end
 
-    it 'unescapes the xml before sending it to receive_salmon' do
+    it 'unescapes the xml before sending it to receive' do
```

Comment:
Weird, I ran `bin/pronto run -c upstream/develop` and it didn't give me anything :P